### ISSUE TITLE
Is it possible to use mtk bypass from an Android device with termux or an apk on another Android device currently in bootloop 

### ISSUE THREAD

**Issue by Xinul-Hacker** [MTK-bypass/bypass_utility#87]
My question is whether it is currently possible to use the mtk bypass from a rooted Android device with termux or some other program?
If not, how difficult would it be to port mtk_bypass to Android termux??

**Comment by SpatMatizm** [comment]
Yes actually, I was able to run it using a patched libusb, even without root, but since you have root, it wouldn't be a problem for you, if you still want I'll give you a tutorial, but this issue is really old tho, don't know if you're even alive :⁠^⁠)

<img width="720" height="1520" alt="Image" src="https://github.com/user-attachments/assets/b64f10a4-4a9d-4435-9a65-4722f6c5b9e6" />
